File: cognizent_INT/Factorial.cs
using System;
public class Fibbonachhi
{
      public static void Main(String[] args)
      {
            Console.WriteLine("Enter Limit: ");
            int num=int.Parse(Console.ReadLine());
            int fact=1;
            for(int i=1;i<=num;i++)
            {
                  fact*=i;
            }
                  Console.WriteLine("Factorial of "+num+" is "+fact);
      }
}

File: cognizent_INT/Factorial.java
import java.util.Scanner;

public class Factorial {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter Limit: ");
        int num = Integer.parseInt(scanner.nextLine());
        int fact = 1;
        for (int i = 1; i <= num; i++) {
            fact *= i;
        }
        System.out.println("Factorial of " + num + " is " + fact);
    }
}
